packages/google_sign_in/example/android/app/build.gradle

```diff
@@ -33,6 +33,9 @@ android {
             // Signing with the debug keys for now, so `flutter run --release` works.
             signingConfig signingConfigs.debug
         }
+        profile {
+            matchingFallbacks = ['debug', 'release']
```

Comment:
This is not necessary with Flutter master where it is part of the `flutter.gradle` script in Flutter tools.
We should either not do that here (just wait for alpha to roll), or do it on all plugins.

Reply:
A fix of alpha roll blocker https://github.com/flutter/flutter/issues/13742 is ready, so an alpha roll should be just around the corner.